tests/transforms/test_rand_simulate_low_resolution.py

```
@@ -78,6 +81,41 @@ def test_value(self, arguments, image, expected_data):
+    def test_thread_safety(self):
+        # Verify that concurrent calls do not corrupt each other's track_meta state
+        # (regression test for GitHub issue #8409).
+        errors = []
+
+        def run_transform():
+            img = torch.ones(1, 4, 4, 4)
+            tfm = RandSimulateLowResolution(prob=1.0, zoom_range=(0.5, 0.6))
+            before = get_track_meta()
+            try:
+                tfm(img)
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)